components/Player/Pages/playerMatchesColumns.jsx

```diff
@@ -1,20 +1,18 @@
-import React from 'react';
 // import { Link } from 'react-router';
-import ActionDone from 'material-ui/svg-icons/action/done';
 import { transformations } from '../../../utility';
 
 export default [{
-  displayName: 'ID',
-  field: 'match_id',
-  width: 2,
-  sortFn: true,
-  displayFn: transformations.match_id,
-}, {
   displayName: 'Hero',
   field: 'hero_id',
-  width: 3,
+  width: 3.1,
   displayFn: transformations.hero_id,
 }, {
+  displayName: 'Id',
+  field: 'game_mode',
+  width: 1.8,
```

Comment:
I agree this should sort on match ID.  You can do this by either changing the field, or the sortFn